backend/Dockerfile

```diff
@@ -4,6 +4,9 @@ WORKDIR /usr/src/app
 
 COPY . .
 
+# Copy the rabbitmq_client_package into the container
+COPY ../packages/Akvo_rabbitmq_client /app/Akvo_rabbitmq_client
```

Comment:
I think the name of the package could be `RagDollQueueManager` instead of `Akvo_rabbitmq_client`. What do you think @kjkoster ?

Reply:
or RagDollMQ

Reply:
1 vote for AkvoMQ